dxm/src/main/java/io/pcp/parfait/dxm/PcpOffset.java

```diff
@@ -5,4 +5,5 @@
 
 interface PcpOffset {
```

Comment:
I realise that `length` is fairly weak as a name too.  I'm trying to get to grips with the purpose of the class.

It's sort of a `MemoryRange` or, Segment/Slot. A location + a length to define the range within a block?